src/Consumable.php

```diff
@@ -188,6 +201,29 @@ public function out($ID, $itemtype = '', $items_id = 0)
                 ]
             );
             if ($result) {
+                switch ($itemtype) {
+                    case 'User' :
+                        $model = new User();
+                        $model->getFromDB($items_id);
+                        $name = strtoupper($model->fields['name']);
+                        break;
+                    case 'Group' :
+                        default:
+                        $model = new Group();
+                        $model->getFromDB($items_id);
+                        break;
+                }
```

Comment:
\`switch` seems useless:
```suggestion
                $item = getItemForItemtype($itemtype);
                $item->getFromDB($items_id);
```